awx/main/tasks/callback.py

```diff
@@ -85,6 +85,8 @@ def event_handler(self, event_data):
         # which generate job events from two 'streams':
         # ansible-inventory and the awx.main.commands.inventory_import
         # logger
+        if event_data.get('event') == 'keepalive':
+            return
```

Comment:
From discussion with @TheRealHaoLiu, we are unlikely to need this, but I would prefer to keep it in anyway. Maybe leave a comment that it should be dead code.